### ISSUE TITLE
Pretrained Models have zero accuracy on Imagenet

### ISSUE THREAD

**Issue by fernande2000** [keras-team/keras#10068]
I have a problem with pretrained models, I tried to debug, watch variables and everything possible for several days, and I still have the same problem. 
I start loading x_train and y_train, which are a list of all training files on imagenet (in the same folder) and the corresponding list of ground truth values, as a list. 
Then I run the relatively short code that you can find here :
 https://gist.github.com/fernande2000/69f57202c391ec08d1552827d7c635b8
to prepare the data and feed it to a pretrained keras  model with imagenet weights.
The problem is that I get an accuracy=0.0 all the time, even if I keep training. 
The same happens if I use one of the batches, using model.evaluate I also get zero accuracy, even if by hand I can count that the decoded prediction and the provided labels and they agree about 30% of the time (this should also be an issue, an accuracy of 0.3 is still very low for  pretrained weights).  Is there any obvious mistakes in the code, that makes it do something unintended? Thanks!!!
UPDATE: After training a few hours accuracy is up to 50%, I will let it a little more and see what it is that it has learned, which could perhaps give me a clue.

**Comment by fernande2000** [comment]
I believe  the mistake is that I used a different list of labels that the one used to train the model, why are there more than a list circulating out there is a mystery to me!
